### PR TITLE
More API docs improvements

### PR DESCRIPTION
**Pull request by dtribble**
docs: Add E references and fix up helpers.

### CODE REVIEW

File: main/zoe/api/zoe-helpers.md [hunk 1]
```diff
@@ -173,9 +173,44 @@ import { makeZoeHelpers } from '@agoric/zoe/src/contractSupport/zoeHelpers';
 
 const { swap } = makeZoeHelpers(zoe);
 
-  const seat = harden({
-    matchOffer: () => swap(firstInviteHandle, inviteHandle),
-  });
+  // `firstOfferHandle` is from a prior offer to the contract
+  const hook = newHandle => swap(firstInviteHandle, newHandle);
```
Comment (katelynsills) [inline main/zoe/api/zoe-helpers.md:177]: We should change any mention of inviteHandle to offerHandle

File: main/zoe/api/zoe-helpers.md [hunk 2]
```diff
@@ -173,9 +173,44 @@ import { makeZoeHelpers } from '@agoric/zoe/src/contractSupport/zoeHelpers';
...
+## zoeHelper.inviteAnOffer(options)
+- `options` `{offerHook, customProperties, expected}` Optional name 
+  properties of the 
+- Returns: a promise for the new inviteHandle
+
+Make an invitation to submit an Offer to this contract. This
+invitation can be given to a client, granting them the ability to
+participate in the contract.
+
+If "offerHook" is provided, it will be called when the invitation is exercised
```
Comment (katelynsills) [inline main/zoe/api/zoe-helpers.md:190]: There's two new terms here - "invitation" and "exercised". I think readers will be confused. We've talked about invites being redeemed (not necessarily better language) so I think we should stick with invite and redeemed for consistency.

File: main/zoe/api/zoe-helpers.md [hunk 3]
```diff
@@ -173,9 +173,44 @@ import { makeZoeHelpers } from '@agoric/zoe/src/contractSupport/zoeHelpers';
...
+and an offer is submitted. The callback will get a reference to the offer.
```
Comment (katelynsills) [inline main/zoe/api/zoe-helpers.md:191]:
```suggestion
and an offer is submitted. The callback will get a reference to the offerHandle.
```

File: main/zoe/api/zoe-helpers.md [hunk 4]
```diff
@@ -173,9 +173,44 @@ import { makeZoeHelpers } from '@agoric/zoe/src/contractSupport/zoeHelpers';
...
+
+If the "expected" option is provided, it should be an {ExpectedRecord}.
+This is like a {Proposal}, but the amounts in 'want' and 'give' should be null,
+and the 'exit' should have a choice but the contents should be null.
+If the client submits an Offer which does not match these expectations,
+that offer will be rejected (and refunded) without invoking the offerHook.
```
Comment (katelynsills) [inline main/zoe/api/zoe-helpers.md:197]: This is great!

File: main/zoe/api/zoe.md [hunk 5]
```diff
@@ -2,20 +2,34 @@
...
-Zoe is a long-lived and well-trusted contract that enforces offer safety for the contracts that use it. Zoe has a single `inviteIssuer` for the entirety of its lifetime. By having a reference to Zoe, a user can get the `inviteIssuer` and thus validate any `invite` they receive from someone else.
-
-## zoe.getInviteIssuer()
+Zoe provides a framework for deploying and working with smart contracts. It is accessed 
+as a long-lived and well-trusted service that enforces offer safety for the contracts that use it. Zoe has a single `inviteIssuer` for the entirety of its lifetime. By having a reference to Zoe, a user can get the `inviteIssuer` and thus validate any `invite` they receive from someone else.
+
+::: tip Zoe is accessed asynchronously
+The Zoe service is accessed asynchronously, using a standards-track library extension
+to JavaScript that uses promises as remote references. In code, the Zoe service instance 
+is refered to via `zoe`, whcih only supports asynchronous invocation. Operations are
```
Comment (katelynsills) [inline main/zoe/api/zoe.md:11]: I think there's typos in this document. Can we run a spell checker on it?

File: main/zoe/api/zoe-helpers.md (hunk 1 above)
Comment (dtribble) [reply main/zoe/api/zoe-helpers.md:177]: Done

File: main/zoe/api/zoe-helpers.md (hunk 2 above)
Comment (dtribble) [reply main/zoe/api/zoe-helpers.md:190]: Done

File: main/zoe/api/zoe.md (hunk 5 above)
Comment (dtribble) [reply main/zoe/api/zoe.md:11]: done

**Pull request merged**